src/userCLI/src/main/kotlin/org/imperial/mrc/hint/userCLI/App.kt

```diff
@@ -36,6 +36,19 @@ fun main(args: Array<String>) {
 
     val dataSource = DbConfig().dataSource(ConfiguredAppProperties())
 
+    fun errorOutputStream(msg: String)
+    {
+        val resources = ResourceBundle.getBundle("ErrorMessageBundle", Locale("en"))
+        try
+        {
+            val message = resources.getString(msg.takeLastWhile { it.isLetter() })
+            System.err.println(message)
+        } catch (e: MissingResourceException)
+        {
+            System.err.println("Could not load ErrorMessageBundle: $msg")
+        }
+    }
```

Comment:
Unknown error causes an issue so I am handling exception here. I am also taking the last word in the initial thrown exception.

Reply:
Why was it trying to take the whole message as the resource key before , but now just the last word? What difference does that make? Are there some errors which do match based on only their last word (as oppsed to only word)?

I think it would be great to add an explanatory comment here, referring to the bug ticket and explaining the last-word usage.